init.sh

```diff
@@ -78,18 +80,20 @@ if [[ ! -f lup-${NAME} ]]; then
     sudo cp lup-${NAME} /etc/sudoers.d/
 fi
 
-if [[ -z "${LIGHTUP_TAR_GZ_VERSION}" ]]; then
-    echo "cloning lightup manager"
-    [[ -d lupmgr ]] || git clone https://github.com/lupfoss/lupmgr.git
-    cd lupmgr && git pull && git checkout ${BRANCH}
-else
-    echo "getting lightup manager"
-    LIGHTUP_TAR_GZ_TARGET="v${LIGHTUP_TAR_GZ_VERSION}.tar.gz"
+if [[ -z "${LIGHTUP_DEBUG}" ]]; then
+    # default production path
+    echo "downloading lightup manager"
+    LIGHTUP_TAR_GZ_TARGET="v${BRANCH}.tar.gz"
     curl -H 'Cache-Control: no-cache' -L https://github.com/lupfoss/lupmgr/archive/refs/tags/"${LIGHTUP_TAR_GZ_TARGET}" --output lupmgr.tar.gz
     tar -xvf lupmgr.tar.gz
     [[ -d lupmgr ]] && rm -rf lupmgr
-    mv lupmgr-"${LIGHTUP_TAR_GZ_VERSION}" lupmgr
+    mv lupmgr-"${BRANCH}" lupmgr
     cd lupmgr
```

Comment:
@manubansal @jeff-lightup I think there is another problem that I think needs to be tackled. We store all our keys in the lupmgr directory. If we delete the lupmgr directory, we will delete the keys. Do we lose something in the process? 

Also, if we do delete the keys, is there some "remnants" that we will need to clean up from the connect server?

Reply:
Also, I don't know if we should overload BRANCH. TAG is not a BRANCH and the BRANCH.tar.gz will not work unless BRANCH is a tag.